Overview.src.html

```diff
@@ -202,6 +202,34 @@ <h2 class="short">Conformance</h2>
 <span>process request body</span> for <var>request</var>.
 <!-- XXX xref "read", "payload body" -->
 
+<h3>Token Binding</h3>
+
+<p>The technique of Token Binding allows user agents to work with servers to protect credentials
+like HTTP cookies.
+User agents and servers mutually agree to use Token Binding by negotiating while setting up
+TLS connections, as described in the
+<a href="https://tools.ietf.org/html/draft-ietf-tokbind-negotiation">Token Binding Negotiation</a>
+spec.
+
+<p>A <dfn title=concept-token-binding>Token Binding</dfn> is established by a user agent generating
+a public-private key-pair for the server and proving possession of the private key on every request
+to the server, as described in
+<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol#section-1">section 1</a>
+of the Token Binding Protocol spec.
+A <dfn title=concept-token-binding-id>Token Binding Id</dfn> is the representation of a
+<span title=concept-token-binding>Token Binding</span>, as described in
+<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol#section-3.2">section 3.2</a>
+of the Token Binding Protocol spec.
+
+<p>A <dfn title=concept-token-binding-message>Token Binding Message</dfn> is transmitted by
+a user agent to a server to prove possession of
+<span title=concept-token-binding>Token Binding</span> keys.
+The value of a <span title=concept-token-binding-message>Token Binding Message</span> is specified
+in <a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol#section-3">section 3</a>
+of the Token Binding Protocol spec.
+A server uses the <span title=concept-token-binding-message>Token Binding Message</span> received
+from a user agent to bind credentials that it issues to that user agent,
+and to verify that credentials presented by the user agent are correctly bound.
```

Comment:
I'd suggest defining the `Sec-Token-Binding` header here instead of defining "token binding message". Something like:

> ```
> The `<dfn http-header><code>Sec-Token-Binding</code></dfn>` header is used to transmit
> token binding metadata from the user agent to the server. Its <a for="header">value</a> is a
> base64url-encoded string [[!RFC4648]]:
>
> <pre>
> Sec-Token-Binding       = 1*( ALPHA / DIGIT / "-" / "_" ) *2( "=" )
> </pre>

I'd also suggest moving the value generation up from inside HTTP-network-or-cache fetch, and into this section. Something like:

> ```
> To <dfn>add token binding metadata to <var>request</var></dfn>, run the following steps:
>
> 1. If ...
> ```

We'll get to the `...` bit down below where the text currently lives.